ca/keyutils/keyutils.go

```diff
@@ -0,0 +1,80 @@
+package keyutils
```

Comment:
It actually supports both keys, as it uses `cfssl/helpers` internally to parse decrypted PEMs